static/js/datatables_config.js

```diff
@@ -0,0 +1,98 @@
+const gen_datatables_config = (overwrites) => {
+  const column_selector = (idx, data, node) => {
+    // https://datatables.net/forums/discussion/42192/exporting-data-with-buttons-and-responsive-extensions-controlled-by-column-visibility
+    if ($(node).hasClass('noVis')) {
+      return false;
+    }
+    const tableID = $(node).closest('table').attr('id');
+    return tableID === undefined? false : $(tableID).DataTable().column(idx).visible();
+  };
+  return Object.assign({
+    dom: 'Bfrtipl',
+    responsive: true,
+    colReorder: true,
+    deferRender: true,
+    createdRow: (row) => {
```

Comment:
Are you sure? PyCharm doesn't agree for some reason, but it may be just a linter bug
![image](https://user-images.githubusercontent.com/1517255/76153197-28f41800-60c9-11ea-82a4-e8cd97dfad4f.png)

Reply:
It's necessary for the points popup to work because datatables recreates the dom element after calling the render function with the display type.
https://datatables.net/reference/option/createdRow

Reply:
Okay, so it's just the linter being stupid and not realizing that this is actually used :man_facepalming: